src/lib/components/SidesFooter.svelte

```diff
@@ -0,0 +1,17 @@
+<div class="mb-6 flex h-auto w-auto justify-between font-inter">
+    <div class="w-48 text-left">
+        <p class="mb-4 max-w-fit text-lg">
+            UP CSI is a service-oriented organization for student learning
+        </p>
+        <p class="underline">lino@up-csi.org</p>
+        <p>+63999999999</p>
+    </div>
+    <div class="flex flex-col text-right underline">
+        <a href="https://www.facebook.com/upcsi">Facebook</a>
+        <a href="https://twitter.com/officialupcsi">Twitter</a>
+        <a href="https://www.linkedin.com/company/up-center-for-student-innovations/">LinkedIn</a>
+        <a href="https://github.com/up-csi">GitHub</a>
+
```

Comment:
Just a small nitpick: we can remove the extra space here. Don't worry, I can remove this on your behalf.
```suggestion
```